### PR TITLE
Maha worker reporting proto

### PR DESCRIPTION
**Pull request by pranavbhole**
@panditsurabhi

### CODE REVIEW

File: worker/src/main/proto/MahaCustomReportRequest.proto [hunk 1]
```diff
@@ -0,0 +1,64 @@
...
+    optional bool dryrun = 7;
+    optional QueueType queueType = 8;
+    optional string userId = 9;
+    optional bool dualWrite = 10;
+    optional string registry_name = 11;
+}
+
+enum DeliveryMethod {
+    EMAIL = 0;
+    MOBSTOR_AND_API = 1;
```
Comment (panditsurabhi) [inline worker/src/main/proto/MahaCustomReportRequest.proto]: Let's rename MOBSTOR_AND_API to CONTENT_STORE or something like that

File: worker/src/main/proto/MahaCustomReportRequest.proto [hunk 2]
```diff
@@ -0,0 +1,64 @@
...
+    JSON = 1;
+}
+
+enum ReportType {
+    AD_HOC = 0;
+    SCHEDULED = 1;
+}
+
+enum QueueType {
+    CMS = 0;
```
Comment (panditsurabhi) [inline worker/src/main/proto/MahaCustomReportRequest.proto]: Let's rename CMS to PULSAR

File: worker/src/main/proto/MahaCustomReportRequest.proto (hunk 1 above)
Comment (pranavbhole) [reply worker/src/main/proto/MahaCustomReportRequest.proto]: done

File: worker/src/main/proto/MahaCustomReportRequest.proto (hunk 2 above)
Comment (pranavbhole) [reply worker/src/main/proto/MahaCustomReportRequest.proto]: done

**Pull request merged**